codalab/apps/web/static/js/worksheet/worksheet_side_panel.js

```diff
@@ -576,7 +582,7 @@ var FileBrowserItem = React.createClass({
                 <td>
                     <div className={this.props.type} onClick={this.props.type != 'file' ? this.browseToFolder : null}>
                         <span className={icon} alt="More"></span>
-                        <a href={this.props.type == 'file' ? file_link : null} target="_blank">{this.props.index}</a>
+                        <a href={this.props.type == 'file' ? file_link : null} target="_blank" className='sidepanel-file-viewer-contents'>{this.props.index}</a>
```

Comment:
I believe `className` attribute should be `class`? Unless this is React specific